nipype/interfaces/cmtk/nx.py

```diff
@@ -289,11 +289,6 @@ def compute_singlevalued_measures(ntwk, weighted=True, calculate_cliques=False):
     iflogger.info("Computing single valued measures:")
     measures = {}
     iflogger.info("...Computing degree assortativity (pearson number) ...")
-    try:
-        measures["degree_pearsonr"] = nx.degree_pearsonr(ntwk)
-    except AttributeError:  # For NetworkX 1.6
-        measures["degree_pearsonr"] = nx.degree_pearson_correlation_coefficient(ntwk)
```

Comment:
I think we want to keep this one. And below, we'll want to keep `degree_assortativity_coefficient`.

Reply:
why keep it if it's just for `=networkx-1.6`, when we are no longer supporting `<networkx-2.0`?

Reply:
I checked, and that's the newer of the two.

Reply:
@effigies sorry, I don't understand. 1.6 is greater than 2.0?

Reply:
No, the `degree_pearsonr` was the function name in `<1.6`. `degree_pearson_correlation_coefficient` is in `>=1.6`.

I agree it seems backwards. I usually put the deprecated version in the `except` block.

Reply:
> I agree it seems backwards. I usually put the deprecated version in the except block.

Ah, yes, that was what confused me. Thank you for checking!